docker/graalvm/Dockerfile

```
@@ -1,5 +1,5 @@
 
-FROM ghcr.io/graalvm/graalvm-ce:ol7-java11-21.0.0
+FROM ghcr.io/graalvm/graalvm-ce:ol7-java11-21.3
```

```suggestion
FROM ghcr.io/graalvm/graalvm-ce:ol7-java11
```
(Optional) - This will keep us on the most recent Java 11 GraalVM